src/components/ListComponent/ListComponent.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ListComponent.css';
+
+function ListComponent({ item }){
+  return (
+    <div>
+        <h3>{ item.title }</h3>
+        <p key={`document_${item.id}`}>
+          { item.body }
```

Comment:
Please read https://github.com/airbnb/javascript/tree/master/react

```
// bad
<Foo bar={ baz } />

// good
<Foo bar={baz} />
```

Reply:
no, this is actually okay as it's not a prop passing but JSX rendering